### PR TITLE
updating names list for notifications

### PR DESCRIPTION
**Pull request by katrinamariehh**
Am I missing anyone?

### CODE REVIEW

**Comment by katrinamariehh** [comment]
ok! I think I have everyone now?

File: src/election_notification_works/handlers/util.clj [hunk 1]
```diff
@@ -101,26 +101,32 @@
...
    "Junie"
    "Kathryn"
    "Katrina"
-   "Li"
+   "Leila"
+   "Liz"
+   "Mariah"
    "Mike"
    "Mike"
+   "Mike"
```
Comment (gws) [inline src/election_notification_works/handlers/util.clj]: Excited for the inevitable "Mike, Mike, Mike" :joy:

Comment (katrinamariehh) [reply src/election_notification_works/handlers/util.clj]: 😹

Comment (katrinamariehh) [reply src/election_notification_works/handlers/util.clj]: 🎤

**Pull request merged**